TCAT/Controllers/RouteOptionsViewController+Extensions.swift

```diff
@@ -117,7 +117,7 @@ extension RouteOptionsViewController: DatePickerViewDelegate {
             buttonTapped = "Leave At Tapped"
         }
 
-        routeSelection.setDatepicker(withDate: date, withSearchTimeType: searchTimeType)
```

Comment:
Ignore this. It'll get moved later (probably using delegates? haven't really thought too much into it yet)

Reply:
can just use a `configure` function probably